tests/docker_mock.py

```diff
@@ -387,8 +385,7 @@ def remove_volume(volume_name):
              .and_raise(docker.errors.APIError, "xyz", response))
 
     if remember_images:
-        global mock_images
```

Comment:
effectively this still works like global variable, it just bypassed warning and obfuscated code. I'm not sure if this worth fixing.

Reply:
Ok, reverted and disabled the warning